Archs/MIPS/MipsOpcodes.cpp

```diff
@@ -104,6 +105,8 @@ const tMipsOpcode MipsOpcodes[] = {
 	{ "lwc1",	"T,(s)",			MIPS_OP(0x31),			MA_MIPS1,	0 },
 	{ "l.s",	"T,i16(s)",			MIPS_OP(0x31),			MA_MIPS1,	0 },
 	{ "l.s",	"T,(s)",			MIPS_OP(0x31),			MA_MIPS1,	0 },
+	{ "lwc2",	"t,i16(s)",			MIPS_OP(0x32),			MA_PSX,	0 },
+	{ "lwc2",	"t,(s)",			MIPS_OP(0x32),			MA_PSX,	0 },
```

Comment:
These are standard MIPS 1 opcodes, so they should be made available on all architectures that don't replace it with something else (i.e. MIPS1 except PSP).